pages/developers/guidelines.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Package requirements and guidelines
+permalink: /guidelines
```

Comment:
This should be nested, too, in the dev folder, like the other dev pages

Reply:
You mean it should be `permalink: /developer/guidelines`? About to commit the correction.